counter-sink/src/main/java/org/springframework/cloud/stream/module/metrics/CounterSinkProperties.java

```diff
@@ -42,6 +42,19 @@
 	 */
 	private String nameExpression;
 
+	/**
+	 * The name of a backend used to store the counter.
+	 */
+	// Stored as a String to allow forward extension of the module
+	private String backend = "memory";
```

Comment:
"backend" is rather general, maybe "store" would be better? taking cue from the README and the JavaDoc which both emphasize this determines where the counts are *stored*, so that intuitively appears to the most self-describing name to use